source/operations/server-side-encryption/configure-minio-kes-aws.rst

```diff
@@ -242,20 +242,22 @@ MinIO specifically requires the following AWS settings or configurations:
 Configuration Reference for AWS Root KMS
 ----------------------------------------
 
-The following section describes each of the |KES-git| configuration settings for
-using AWS Secrets Manager and AWS KMS as the root Key Management Service
-(KMS) for |SSE|:
+The following section describes each of the |KES-git| configuration settings for using AWS Secrets Manager and AWS KMS as the root Key Management Service (KMS) for |SSE|:
```

Comment:
It's odd to define the KMS abbr after already using it.

Maybe:
> ... using AWS Secrets Manager and AWS Key Management System (KMS) as the root KMS for |SSE|:

Reply:
Or define the KMS acronym earlier in the doc so that we don't run into the issue of thinking KMS _always_ means Amazon's KMS.

Reply:
Took a middle path.